kong/tools/string.lua

```diff
@@ -20,18 +22,185 @@ local CR_BYTE    = byte("\r")
 local _M = {}
 
 
-_M.join = pl_stringx.join
+_M.join = require("pl.stringx").join
```

Comment:
Why not `pl_stringx`?

Reply:
Because `pl.stringx` is only used there in two places, and I prefer not being used anywhere else.